### ISSUE TITLE
Missing calendar header

### ISSUE THREAD

**Issue by imWildCat** [roomorama/Caldroid#274]
![qq20150531-1 2x](https://cloud.githubusercontent.com/assets/2396817/7902381/be0ebf5e-07ea-11e5-9e37-f734e5a77ca8.png)

I set up Caldroid in my app by the following code, but it didn't work fine.

```java
void setUpCalendar() {
        vCaldroidFragment = new CaldroidFragment();
        Bundle args = new Bundle();
        Calendar cal = Calendar.getInstance();
        args.putInt(CaldroidFragment.MONTH, cal.get(Calendar.MONTH) + 1);
        args.putInt(CaldroidFragment.YEAR, cal.get(Calendar.YEAR));
        args.putInt(CaldroidFragment.THEME_RESOURCE, com.caldroid.R.style.CaldroidDefaultDark);
        vCaldroidFragment.setArguments(args);

        FragmentTransaction t = getActivity().getSupportFragmentManager().beginTransaction();
        t.replace(R.id.calendar_view_holder, vCaldroidFragment);
        t.commit();
    }
```

Update:

`vCaldroidFragment.getMonthTitleTextView()` returns null.

**Comment by AmmadAliSynnApps** [comment]
Hi,
Have you resolved this issue?
I am also facing same problem when loads CaldroidFragment

**Comment by AmmadAliSynnApps** [comment]
I have added some items to AppTheme style to make similar to CaldroidDefault in caldroid project.
<item name="styleCaldroidViewLayout">@style/CaldroidDefaultCalendarViewLayout</item> <item name="styleCaldroidLeftArrow">@style/CaldroidDefaultLeftButton</item> <item name="styleCaldroidRightArrow">@style/CaldroidDefaultRightButton</item> <item name="styleCaldroidMonthName">@style/CaldroidDefaultMonthName</item> <item name="styleCaldroidGridView">@style/CaldroidDefaultGridView</item> <item name="styleCaldroidNormalCell">@style/CaldroidDefaultNormalCell</item> <item name="styleCaldroidSquareCell">@style/CaldroidDefaultSquareCell</item> <item name="styleCaldroidWeekdayView">@style/CaldroidDefaultWeekday</item>

**Comment by thomasdao** [comment]
This is fixed in latest version 2.1.0

**Comment by pellyadolfo** [comment]
I came to this bug because I have the same issue. In my case is working properly in a device running Android Kitkat (Build.VERSION.SDK_INT: 19). For another device running Lollipop (Build.VERSION.SDK_INT: 21) works perfectly. I tried version 2.1.0 and same thing happens.

I opened an issue here http://stackoverflow.com/questions/32282065/add-buttons-to-caldroid with a test case
